.github/workflows/upstream.yml

```diff
@@ -55,8 +55,72 @@ jobs:
         run: source continuous_integration/scripts/install.sh
 
       - name: Run tests
+        id: run_tests
```

Comment:
@wreis I'm wondering why this was added. From what I can tell I don't think it's used anywhere, but I could be missing something. FWIW this also seems harmless, so I don't mean for this comment to be blocking

Reply:
@jrbourbeau this is a left over from previous version where I used the id as ref for this step, it is harmless yeah and might be useful in future.